testing/mopidy/mopidy.post-install

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+
+adduser -S -D -h /var/lib/mopidy -s /sbin/nologin -G audio mopidy 2>/dev/null
+
+chown -R mopidy:audio /etc/mopidy/extensions.d 2>/dev/null
+chown -R mopidy:audio /var/cache/mopidy 2>/dev/null
+chown -R mopidy:audio /var/lib/mopidy 2>/dev/null
```

Comment:
the permissions should also be set in APKBUILD. sorry for not being clear about that last time.

````sh
pkgusers="mopidy"
pkggroups="audio"
...
package() {
    install -dm755 -o modpidy -g audio "${pkgdir}/var/cache/mopidy"
    ....
}
````

Reply:
This combined with adduser in `pre-install` just causes "unknown user 'mopidy'"...

Reply:
Ah I missed the `pkgusers` and `pkggroups` part. Thanks, that did it, will update the PR.

Reply:
Hmm, now the directories are owned by `messagebus:audio`? I never used that user anywhere, not sure why this is happening.